checkout/src/main/java/com/checkout/tokenization/repository/TokenRepositoryImpl.kt

```diff
@@ -122,6 +126,7 @@ internal class TokenRepositoryImpl(
                 launch(Dispatchers.Main) {
                     when (tokenResult) {
                         is TokenResult.Success -> {
+                            riskSdkUseCase.execute(TokenResult.Success(tokenResult.result.token))
```

Comment:
nit:
invoke the riskSdkUseCase after handling the result for the user; is better to send the log after the handling is launched